app.js

```diff
@@ -1,14 +1,38 @@
 const express = require('express');
+const bodyparser = require('body-parser');
+
 const app = express();
 const port = process.env.PORT || 5000;
 
+app.use(express.json());
+app.use(express.urlencoded({extended:false}));
+
 app.get('/', function(req, res){
   res.send("hello world");
```

Comment:
change to a more appropriate description of project
```suggestion
  res.send("Africa's Talking Technical Interview, see github README.md for more info");
```